data_steward/cdr_cleaner/cleaning_rules/drop_duplicate_states.py

```diff
@@ -0,0 +1,92 @@
+"""
+ Currently there are some individuals with multiple state records from consent.
+ We need to include only one state, dropping all but the most recent record.
+"""
+import logging
+import os
+
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+import sandbox
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
Does this module need the the LOGGER and logging module?  This module isn't logging anything.